---
abstract: "G-quadruplex (G4), a non-classical deoxyribonucleic acid structure, is widely distributed in the genome and involved in various
biological processes. In vivo, high-throughput sequencing has indicated that G4s are significantly enriched at functional regions in a celltype-specific manner. Therefore, the prediction of G4s based on computational methods is necessary instead of the time-consuming
and laborious experimental methods. Recently, G4 CUT&Tag has been developed to generate higher-resolution sequencing data than
ChIP-seq, which provides more accurate training samples for model construction. In this paper, we present a new dataset construction
method based on G4 CUT&Tag sequencing data and an XGBoost prediction model based on the machine learning boost method. The
results show that our model performs well within and across cell types. Furthermore, sequence analysis indicates that the formation
of G4 structure is greatly affected by the f lanking sequences, and the GC content of the G4 f lanking sequences is higher than non-G4.
Moreover, we also identified G4 motifs in the high-resolution dataset, among which we found several motifs for known transcription
factors (TFs), such as SP2 and BPC. These TFs may directly or indirectly affect the formation of the G4 structure"
authors:
- Yizhi Cui
- "..."
- admin
- "..."
- Ruijun Liu
date: "2023-02-13T00:00:00Z"
doi: "https://doi.org/10.1093/bfgp/elad024"
links:
featured: true
image:
  caption:
  focal_point: ""
  preview_only: false
projects:

publication: 'Briefings in Functional Genomics'
publication_short: ""
publication_types:
- "2"
publishDate: "2023-02-13T00:00:00Z"
slides:
summary: 
tags:
- published
title: "Prediction of strand-specific and cell-type-specific G-
quadruplexes based on high-resolution CUT&Tag data"
url_code: ""
url_dataset: ""
url_pdf: ""
url_poster: ""
url_project: ""
url_slides: ""
url_source: ""
url_video: ""
---
